utils/theme/globals.scss

```diff
@@ -10,6 +10,16 @@ body {
   margin: 0;
 }
 
+#__next {
```

Comment:
I cannot tell you how many times I tried to use this selector and it never worked 😒 

glad it's working for you though!